jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1049,17 +1049,28 @@ properties:
             type: string
             enum:
               # FGS in parallel with NIRCam and NIRISS
-             [FULL, FULLBOX, INTRAMODULE, INTRAMODULEBOX, INTRAMODULEX,
-              INTRASCA, SUBARRAY_DITHER, WFSC, AMI, IMAGING, MIMF, WFSS,
+              # shared with NIRCam: FULL, FULLBOX, INTRAMODULE, INTRAMODULEBOX,
+              #   INTRAMODULEX, INTRASCA, WFSC,
+              # shared with NIRISS: AMI, IMAGING, MIMF, WFSS,
+             [SUBARRAY_DITHER,
+
+              # common to NIRISS and FGS in parallel with NIRCam and NIRISS
```

Comment:
Thanks @hbushouse for looking over the proposed changes. I added comments in the form you suggested so common modes are separated. I also added comments listing the shared modes alongside exclusive modes so hopefully all modes for a given instrument are grouped.